lms/templates/js/discussion/discussion-home.underscore

```diff
@@ -0,0 +1,59 @@
+<div class="discussion-article blank-slate">
+  <section class="home-header">
+    <span class="label"><%- gettext("DISCUSSION HOME:") %></span>
+    <% if (window.course_name) { %>
+      <h1 class="home-title"><%- window.course_name %></h1>
+    <% } %>
+  </section>
+
+  <% if (window.ENABLE_DISCUSSION_HOME_PANEL) { %>
+    <span class="label label-settings">
+      <%- gettext("How to use {platform_name} discussions").replace("{platform_name}", window.PLATFORM_NAME) %>
```

Comment:
I changed this PR to use `interpolate` instead of `replace` or `interpolate_text`.